src/python_framework_v02/troute/nhd_io.py

```diff
@@ -161,6 +163,39 @@ def get_ql_from_wrf_hydro(qlat_files, index_col="station_id", value_col="q_later
 
     return ql
 
+def get_usgs_from_wrf_hydro(routelink_file,usgs_timeslices_folder,file_name):
+
+    ds = xr.open_dataset("../../test/input/geo/routelink/routeLink_subset.nc")
+    df = ds.to_dataframe()
+    df = df.reset_index()
+    df.drop(df.columns.difference(['link','to','gages','ascendingIndex']), 1, inplace=True)
+    df['gages'] = df['gages'].str.decode('utf-8')
+    nan_value = float("NaN")
+    df['gages'].replace('               ', nan_value, inplace=True)
```

Comment:
This operation coerces gages to a float dtype for the column. Is that intended?

Reply:
Yes, it was loading in with extra characters 'b in front of every value.